lib/testmap.py

```diff
@@ -83,7 +83,6 @@ def contexts(image: str, *scenarios: Iterable[str], repo: str | None = None) ->
         ],
         '_manual': [
             'centos-9-bootc',
-            'rhel-9-5',
```

Comment:
Is this intentional? Everywhere else you remove 9-6.

Reply:
It is intentional, yes. We've traditionally not tested starter-kit on rhel, but I'd like to keep the latest rhel-9-Y in manual. I.e. read this as "move manual from 9-5 to 9-6". Sorry for the confusion.

Reply:
Alright!